docs/h2t-scc.md

```diff
@@ -0,0 +1,187 @@
+# Troubleshoot Security Context Constraints (SCC)
+
+This document contains information regarding troubleshooting SCC related issues. Document includes the initial diagnosis step which helps find the actual error. To solve the SCC related issues need a basic understanding of the SCC and this is done by explaining what is SCC is along with its role in the cluster. 
```

Comment:
```suggestion
This document contains information regarding troubleshooting SCC related issues. 
Document includes the initial diagnosis step which helps find the actual error. 
To solve the SCC related issues, you need a basic understanding of the SCC, and we will explain what SCC is along with its role in the cluster. 
```
One sentence per line (for easier updates later), and minor grammatical changes.

Reply:
Modified paragraph one sentence per line, But page view still showing as paragaraph. If i give space for each line, then able to see the publish page as one sentence per line

Reply:
The "one sentence per line" policy is for the .md file, not for the final rendering for readers to see. It's only for the source code (.md) so when we update something in a sentence, the PR would show only that line as different, and it's clear that it doesn't touch/modify any other sentences, makes it easier to review, I think. Thoughts?